src/freertos_drivers/pic32mx/usb_cdc_device_descriptors.c

```diff
@@ -0,0 +1,332 @@
+/*******************************************************************************
```

Comment:
Hmm...  Could we add this license note to the list of license notes in the doc/main.dox so that it does not get lost in the sea of code?

Reply:
done